libs/admin/src/lib/components/user-edit/user-edit.component.html

```diff
@@ -1,44 +1,102 @@
-<form [formGroup]="formGroup">
-  <div class="edit-area">
-    <div *ngIf="user" class="header">
-      <div class="email">
-        {{ user.email }}
+<mat-card class="card">
+  <div class="user-edit">
+    <form [formGroup]="formGroup" class="form">
+      <div *ngIf="user" class="header">
+        <h2 class="email">
+          {{ user.email }}
+        </h2>
       </div>
-    </div>
-    <div class="label-heading label-heading--before-form-field">Sprache</div>
-    <mat-form-field class="lang-form-field">
-      <mat-label>Sprache</mat-label>
-      <mat-select formControlName="lang">
-        <mat-option value="de">DE</mat-option>
-        <mat-option value="en">EN</mat-option>
-        <mat-option value="fr">FR</mat-option>
-        <mat-option value="it">IT</mat-option>
-        <mat-option value="rm">RM</mat-option>
-      </mat-select>
-    </mat-form-field>
+      <div class="user-data">
+        <div class="user-data__group">
+          <mat-form-field class="lang-form-field" subscriptSizing="dynamic">
+            <mat-label translate>admin.userPage.lang</mat-label>
+            <mat-select formControlName="lang">
+              <mat-option value="de" translate><span translate>admin.languages.de</span></mat-option>
+              <mat-option value="en" translate><span translate>admin.languages.en</span></mat-option>
+              <mat-option value="fr" translate><span translate>admin.languages.fr</span></mat-option>
+              <mat-option value="it" translate><span translate>admin.languages.it</span></mat-option>
+              <mat-option value="rm" translate><span translate>admin.languages.rm</span></mat-option>
+            </mat-select>
+          </mat-form-field>
+        </div>
+        <div class="user-data__group" *rxLet="isCurrentUser$; let isCurrentUser">
+          <mat-checkbox formControlName="isAdmin" color="primary">
+            <span translate>admin.userPage.admin</span>
+          </mat-checkbox>
+        </div>
+      </div>
+      <div class="workgroups">
+        <div class="workgroups__header">
+          <h3 translate>admin.workgroups</h3>
+          <form>
+            <mat-form-field>
+              <mat-label translate>admin.userPage.addWorkgroups</mat-label>
```

Comment:
After adding a Workgroup to a User, the List gets filtered by the added workgroup
Only after clicking the X, the other workgroups appear again
Selecting a Workgroup should reset the select component to its initial look